src/components/organisms/FileTreePane/TreeItem.tsx

```diff
@@ -70,13 +73,24 @@ const TreeItem: React.FC<TreeItemProps> = props => {
   } = props;
 
   const [isTitleHovered, setTitleHoverState] = useState(false);
+  const [remotePathUrl, setRemotePathUrl] = useState<undefined | string>();
 
   const fileOrFolderContainedInFilter = useAppSelector(state => state.main.resourceFilter.fileOrFolderContainedIn);
   const fileMap = useAppSelector(state => state.main.fileMap);
   const osPlatform = useAppSelector(state => state.config.osPlatform);
   const selectedPath = useAppSelector(state => state.main.selectedPath);
   const resourceMap = useAppSelector(state => state.main.resourceMap);
   const helmValuesMap = useAppSelector(state => state.main.helmValuesMap);
+  const git = useAppSelector(state => state.git);
+  const selectedProjectRootFolder = useAppSelector(state => state.config.selectedProjectRootFolder);
+  const remotePath = useMemo(
+    async () =>
+      selectedProjectRootFolder
+        ? promiseFromIpcRenderer('git.getGitRemoteUrl', 'git.getGitRemoteUrl.result', selectedProjectRootFolder)
```

Comment:
I would like the remote to be stored directly in the redux store on the `GitRepo` model
In the `electron/app/git.ts` file you'll find the `getGitRepoInfo` method where the `gitRepo` object is initialized.
There, you could call the `getGitRemoteUrl` method that you've created and set the remote to the `gitRepo` object.
You'll have to add the new property on the GitRepo type as well.

Reply:
yep feels cleaner thanks mate 👍

Reply:
updated like this 👍